### PR TITLE
7793 - Editor Text Header and Color

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This pull request will fix editor text not changing to other font headers after changing colors.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/7793

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/editor/example-index.html
- Select all the text inside the editor
- Change to `Header 1`
- Change the color of the text
- Change to `Normal Text`

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — CHANGES_REQUESTED** [review]
@tjamesallen15 the only issue i see with the steps is if you notice the color is not changed.

- select all
- change to header 1
- change color to red
- change to normal text
- color is not red

**Comment by jbrcna** [comment]
you can now changed the heading style after changing font color. However, when color is changed and you switch the heading style, the font color switches to black


https://github.com/infor-design/enterprise/assets/56722184/09aae0db-be55-4a93-bd2a-cd60d97d3306

**Comment by ericangeles** [comment]
@tjamesallen15 can you take a look at @jbrcna's comment?

**Comment by tmcconechy** [comment]
@tjamesallen15 @ericangeles i think me and @jbrcna are seeing the same thing. The color didnt change in some cases (or changed back)

**Comment by tjamesallen15** [comment]
Can we test this on local @jbrcna @tmcconechy,

For some reason it, my changes doesn't reflect on branch link

**Review by jbrcna — DISMISSED** [review]
this is now working In local.
http://localhost:4000/components/editor/example-index.html
https://github.com/infor-design/enterprise/assets/56722184/24394b49-8c06-4959-aabf-651357167bb8

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -5,8 +5,10 @@
 ## v4.88.0 Fixes
 
 - `[Breadcrumb]` Updated hover color for breadcrumb in header. ([#7801](https://github.com/infor-design/enterprise/issues/7801))
+- `[Chart]` Fix on focus border being off in chart legend items. ([#7850](https://github.com/infor-design/enterprise/issues/7850))
```
Comment (ericangeles) [inline docs/CHANGELOG.md:8]: Can you remove the duplicate entry log, @tjamesallen15 ? It's the #7850

**Review by tmcconechy — APPROVED** [review]

**Comment by tmcconechy** [comment]
Artem is looking when the resource dont reflect its usually a resources problem on the server

**Pull request merged**